infrastructure/terraform/README.md

```diff
@@ -60,7 +42,45 @@ variable "doks_k8s_version" {
     cp bootstrapper.tfvars.sample bootstrapper.tfvars
     ```
 
-4. Open the `bootstrapper.tfvars` file and adjust settings according to your needs using a text editor of your choice (preferably with [HCL](https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md) lint support).
+4. Open the `bootstrapper.tfvars` file and adjust settings according to your needs using a text editor of your choice (preferably with [HCL](https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md) lint support). To see the default values, look at `variables.tf`.
+
+     **Important:** 
+    You will need to replace the `do_token` with your own Personal Access Token.
+
+    List Kubernetes versions that can be used with DigitalOcean clusters.
+
+    ```console
+    doctl k8s options versions
+    ```
+    ```bash
```

Comment:
To verify, do you mean to have it be like this?

    ```console
    doctl k8s options versions

    Slug           Kubernetes Version    Supported Features
    1.29.1-do.0     1.29.1                cluster-autoscaler, docr-integration, ha-control-plane, token-authentication
    1.28.6-do.0     1.28.6                cluster-autoscaler, docr-integration, ha-control-plane, token-authentication
    ...
    ```
    
If yes, I changed/split this intentionally because the copy icon copies the text as well and you'll get errors running the whole thing or have to manually highlight just the command. I think the two options that make the most sense here are to either remove the output or to write above it that this is an example output. I don't know that having the output is necessary at all here.

Reply:
@tiffanyfay : yeah remove the console output and just keep the command. Thanks

Reply:
@diabhey Done